CorsixTH/Lua/config_finder.lua

```diff
@@ -18,8 +18,10 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE. --]]
 
+dofile "system/path"
```

Comment:
This still needs to be fixed. When I try to do require instead. I get an error that the module loader can't find it.